src/lightning_lite/accelerators/cpu.py

```diff
@@ -0,0 +1,65 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, List, Union
+
+import torch
+
+from lightning_lite.accelerators.accelerator import Accelerator
+from lightning_lite.utilities import device_parser
+
+
+class CPUAccelerator(Accelerator):
+    """Accelerator for CPU devices."""
+
+    def init_device(self, device: torch.device) -> None:
+        """
+        Raises:
+            ValueError:
+                If the selected device is not CPU.
+        """
+        if device.type != "cpu":
```

Comment:
What if the device is a meta device ?

Reply:
Yes, I only moved the code here because this validation is also in PL. Going to leave it like this for now.